docs/fedora-releases-guide.md

```diff
@@ -18,6 +18,12 @@ Doing Fedora releases with Packit means utilising these jobs:
 
 Every job takes care of a different part of the release process.
 
+Here below a short comparison between `propose-downstream` and `pull-from-upstream`.
+
+![Sync downstream job comparison picture](img/fedora-releases-guide/sync_downstream_summary.png)
+
+![Sync downstream job comparison details](img/fedora-releases-guide/sync_downstream_details.png)
```

Comment:
I know you've spent a lot of time on that, but do you think we can squeeze the info into a regular markdown table?

It might be tricky to do the formatting/spacing/new-lines correctly, but I've tried e.g. `</br>` here, on Github:

| . | propose downstream | pull-from-upstream |
| --- | ------------------------------- |----------------------------- |
| Service |  Version is retrieved from the **upstream project release event** </br> Specfile is retrieved from the **upstream repo** | Version is retrieved from the **https://release-monitoring.org event**</br> Specfile is retrieved from the **dist-git repo**. |
| Packit CLI | ... | ... |

If we make the content searchable and accessible for people with screen readers,... And also the style will match.


(And I know I am the last one who should say this, since I've already prepared a few charts/diagrams for docs in Inkscape and made the same "mistake"..;(

Reply:
Do you mean only the second picture or both of them?